frontend/src/routes/AdminRoute.tsx

```diff
@@ -26,7 +33,7 @@ const AdminRoute: React.FC<Props> = ({ children }: Props) => {
 
     return <Navigate to='/' />;
   }
```

Comment:
like this will work, but probably you want to check whether there is any error first
```
const isUserAdmin = user?.user_type === 'admin';

  if (error) {
    return <ErrorPage />;
  }

  if (isUserAdmin) {
    return <>{children}</>;
  }

  return <Navigate to="/" />;

```

Reply:
This solution doesn't work :pleading_face:
https://github.com/HackYourFuture-CPH/class-25-ironside/assets/33804102/7e5a5837-d237-4a03-a1d7-fd914a8d883d

Reply:
I think it was an issue related to my above 👆 that `user` was null but `currentUser` was not